build-system/eslint-rules/window-property-name.js

```diff
@@ -0,0 +1,127 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview
+ * Checks that custom properties set on the window are prefixed with '__AMP_'.
+ * The prefix is invalid in `id` attributes which prevents DOM clobbering.
+ */
+
+'use strict';
+
+const path = require('path');
+
+// TODO(choumx): Fix extensions code and add 'extensions/' here.
+const PATHS_TO_INCLUDE = ['src/'];
```

Comment:
This breaks when your local `amphtml` client happens to live in a directory like `~/src/amphtml/`. Fix coming up.

Reply:
Fix sent out in #24891.